### PR TITLE
Add dependency version specs

### PR DESCRIPTION
**Pull request by NeonDaniel**
Pins minimum versions for dependencies with stable releases

### CODE REVIEW

**Review by mikejgray — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+ovos_plugin_manager~=0.0.25
+ovos_workshop~=0.0.15
+ovos_utils~=0.0,>=0.0.38
 ovos_classifiers
-pyjokes
-ovos-translate-server-plugin>=0.0.0a5
-neon-lang-plugin-libretranslate
+pyjokes~=0.6
+ovos-translate-server-plugin~=0.0.1
+neon-lang-plugin-libretranslate~=0.2
```
Comment (JarbasAl) [inline requirements.txt]: this is no longer needed, it was a workaround for missing dependencies before the translate server was introduced

File: skill.json [hunk 2]
```diff
@@ -18,13 +18,13 @@
     "systemDeps": false,
     "requirements": {
         "python": [
-            "neon-lang-plugin-libretranslate",
-            "ovos-translate-server-plugin>=0.0.0a5",
+            "neon-lang-plugin-libretranslate~=0.2",
```
Comment (JarbasAl) [inline skill.json]: skill.json now belongs in locale folder and has different fields https://openvoiceos.github.io/ovos-technical-manual/skill_json/

Comment (JarbasAl) [reply skill.json]: https://github.com/OpenVoiceOS/OVOS-skills-store/blob/main/skill-dadjokes.json

Comment (NeonDaniel) [reply skill.json]: automation changes are beyond the scope of this change

Comment (JarbasAl) [reply skill.json]: oops, didnt realize it was a automation commit only looked at global diff

**Review by JarbasAl — APPROVED** [review]

**Comment by NeonDaniel** [comment]
Tests were failing here due to a typo in the changes... I'll send a follow-up PR

**Pull request merged**